src/main/java/mate/academy/springintro/dto/book/BookDto.java

```diff
@@ -20,4 +21,6 @@ public class BookDto {
     private String description;
 
     private String coverImage;
+
+    private List<String> categoryName;
```

Comment:
you should add `categoryIds` instead `names`. Also check your task you have to return BookDtoWithoutCategoryIds in BookController

Reply:
fix comment